cmd/talosctl/cmd/mgmt/cluster/create/clusterops/configmaker/internal/makers/helpers.go

```diff
@@ -25,9 +21,5 @@ func ParseOmniAPIUrl(urlIn string) (*url.URL, error) {
 		return nil, err
 	}
```

Comment:
https://github.com/siderolabs/talos/blob/1cf12ea9cd9f054987edd17f89cb001a0bdfe243/pkg/machinery/config/types/siderolink/siderolink.go#L131-L132

Use this method, please don't do any other validations.

If you want to validate, just put it into the config document and run validate on it